images/cassandra/tests/main.tf

```diff
@@ -0,0 +1,204 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "~> 2.11.0"
+    }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_string" "ref" { input = var.digest }
+
+data "oci_exec_test" "test" {
+  for_each = toset(fileset(path.module, "*.sh"))
+
+  digest = var.digest
+  script = "${path.module}/${each.value}"
+}
```

Comment:
```suggestion
```

I think these k8s tests are good enough where we can just 🪓 this!

Reply:
definitely, thanks a bunch, that script took a lot of time while building but tf was even better so let it go😭